src/transformers/generation/continuous_batching/continuous_api.py

```diff
@@ -604,6 +608,9 @@ def __init__(
             streaming: Whether to stream tokens as they are generated
         """
         self.model = model.eval()
+        attn_implementation = "paged|" + self.model.config._attn_implementation
+        load_and_register_kernel(attn_implementation)
```

Comment:
Not familiar with this, so just resolve if this will work with non-kernelized attn like `paged|sdpa` or `paged|eager`

Reply:
It won't cuz not expected, but yeah I can fix this